docs/technical-guide/developer/ui.md

```diff
@@ -205,6 +205,150 @@ Remember that nesting selector increases specificity, and it's usually not neede
   fill: var(--icon-color);
 }
 ```
+Note: We are using css modules, thanks to that same class on different file don't crash.
+
+### Use CSS logical properties
+
+The [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) define styles relative to the content’s writing mode (e.g., inline, block) instead of physical directions (left, right, etc). This improves support for right-to-left (RTL) languages and enhances layout flexibility.
```

Comment:
Let's add an explicit exception for `width` and `height`

Reply:
Why is there an exception for this? Since we are already using the 'inline' and 'block' concepts everywhere, what benefit is there in not using them in these two specific properties? I think it's harder to remember.
Or at least make it optional.

Reply:
Oh, I see there is already an exception at https://github.com/penpot/penpot/pull/6521/files#diff-dce81c18058cfb49e9a4e987ad629d5bcd08f7e310f52a4db6d14779bca983bcR230